JS/Lesson22/gulcin/accordion.js

```diff
@@ -0,0 +1,14 @@
+const accordionItems = document.querySelectorAll('.accordion__item');
+const accordionItemBodies = document.querySelectorAll('.accordion__item-body');
+
+
+accordionItems.forEach(item => item.addEventListener('click', openAccordion));
+
+function openAccordion(e) {
+    accordionItems.forEach(function (item) {
+        if (item !== e.currentTarget) {
+            item.classList.remove('active');
```

Comment:
In accordion you usually allow a user to have more than one items expanded at the same time so I'd avoid clearing up 'active' class. Instead, just toggle the class for the current element.